res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -640,20 +686,15 @@ PioneerDDJSX.bindNonDeckControlConnections = function(bind) {
     }
 
     engine.connectControl("[AutoDJ]", "enabled", "PioneerDDJSX.autoDJTimer", !bind);
-
-    engine.connectControl("[Channel1]", "VuMeter", "PioneerDDJSX.VuMeterLeds", !bind);
-    engine.connectControl("[Channel2]", "VuMeter", "PioneerDDJSX.VuMeterLeds", !bind);
-    engine.connectControl("[Channel3]", "VuMeter", "PioneerDDJSX.VuMeterLeds", !bind);
-    engine.connectControl("[Channel4]", "VuMeter", "PioneerDDJSX.VuMeterLeds", !bind);
 };
 
 PioneerDDJSX.setDeckSoftTakeover = function(channel, enable) {
```

Comment:
Yes, this is correct. But I think I'm going to remove the handshake and the `setDeckSoftTakeover`, too (because the pot/knob states are initialized by the controller and have to be set by the controller).